src/client/pages/Debugger/ActualDebugger.tsx

```diff
@@ -18,6 +19,8 @@ const Row = styled.div`
 `
 
 export const ActualDebugger: React.FC = () => {
+  const [isCleaningQuoteState, setIsClearningQuoteState] = useState(false)
```

Comment:
Typo 🙈 
```suggestion
  const [isCleaningQuoteState, setIsCleaningQuoteState] = useState(false)
```
Also, I think a better name might be
```suggestion
  const [isResettingAllState, setIsResettingAllState] = useState(false)
```
or
```suggestion
  const [isResettingQuoteState, setIsResettingQuoteState] = useState(false)
```